docs/credible/example--qr-issuance.md

```diff
@@ -0,0 +1,23 @@
+---
+id: example--qr-issuance
+title: qR Code Issuance
+---
+
+## Introduction
+
+This video demonstrates the qR-triggered issuance flow described on the [Core
+Concepts](concepts.md#offer-flow) page.  Note that:
+* `java.theosirian.com` is the test-server generating the qR code and issuing
+  the credential. The pop-up automatically gets consent from the user about any
+  domain referenced from a qR code. This pop-up, which is enabled by default, is
+  to prevent spoofing or phishing attacks, should the page serving the qR and
+  the contents of the qR be in different domains. Further validation can be
+  added as a failsafe.
+* The credential offered, containing a proof of control of a doximity.com
+  account, is dummy data from a client project. The contents displayed should
+  normally come from a processing of the contents of the actual credential, but
+  in this demo, they are mocked up graphically by a few manual overrides in the
+  folder /lib/app/pages/credentials/widget/document/
+
+![credible-vc-vp-half](https://user-images.githubusercontent.com/95347/113591505-f422ff00-9601-11eb-9e16-4d0c45ab8ef9.gif)
```

Comment:
+1 on moving the GIF outside the repo.

Should say `QR` instead of `qR`? https://en.wikipedia.org/wiki/QR_code

> automatically gets consent

Sounds not very consentful...

Reply:
Not sure where I got the strange habit of camel-casing qR code, I always thought that was official but wiki clearly thinks otherwise.  And I've rephrased the consent-override to sound less creepy, good catch!